RandomData.Api/DateTimeGenerators/DateGenerators/RandomDateFormats.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace RandomData.Api.DateTimeGenerators.DateGenerators
+{
+    public enum RandomDateFormats
```

Comment:
You should add [JsonConverter(typeof(JsonStringEnumConverter))] to allow swagger display text instead of numbers